tests/performance/Makefile

```diff
@@ -16,42 +20,29 @@ __check_defined = \
     $(if $(value $1),, \
       $(error Undefined $1$(if $2, ($2))))
 
-.PHONY: requirements
-requirements: ## compiles pip requirements (.in -> .txt)
-	@$(MAKE_C) requirements reqs
-
-
 .env: .env-devel ## creates .env file from defaults in .env-devel
 	$(if $(wildcard $@), \
 	@echo "WARNING #####  $< is newer than $@ ####"; diff -uN $@ $<; false;,\
 	@echo "WARNING ##### $@ does not exist, cloning $< as $@ ############"; cp $< $@)
 
 .PHONY: build
-build:
+build: ## builds distributed osparc locust docker image
 	docker buildx build --tag local/locust:latest .
 
-.PHONY: up
-up:
+.PHONY: up down run
+up: ## runs distributed osparc locust using target=locust_test_file.py, optional host can be set
 	@$(call check_defined, target, please define target file when calling $@ - e.g. ```make $@ target=MY_LOCUST_FILE.py```)
-	@export TARGET=$(target); \
-	docker-compose --file docker-compose.yml up
+	@export LOCUST_FILE=$(target); \
+	export TARGET_HOST=$(if $(host),$(host),"http://$(get_my_ip):9081"); \
```

Comment:
MINOR: TARGET_HOST  is really TARGET_URL?

Reply:
done